packages/server/src/db/index.js

```diff
@@ -121,6 +132,10 @@ async function getUser(id) {
         }
         user.agency.subagencies = subagencies;
     }
+    delete user.tenant_id;
+    delete user.tenant_display_name;
+    delete user.tenant_main_agency_id;
+    delete user.tenant_uses_spoc_process;
```

Comment:
nice catch, would be nice to keep this in mind to delete other temp fields in the future, for example agency_name. Maybe we start using DTOs to hide and transform fields.